dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -30,6 +29,8 @@
  *= require details-polyfill
  */
 
+// TODO: Move into code-studio.scss when it exists.
+@import "/code-studio/assets/video-js/video-js.css";
```

Comment:
Last year, I experimented with extracting video.js into a self-contained js bundle, so it could be cleanly separated from the dashboard application.js. (I abandoned the effort at the time, because of how complicated and time-consuming working with and testing video.js changes ended up being.)

In that work I managed to get a decent system going using the [browserify-css](https://github.com/cheton/browserify-css) plugin, which not only supported inlining css into a browserify bundle, but also supported inlining css relative-url directives according to custom logic. See  the code here: [bundle.js](https://github.com/code-dot-org/code-dot-org/blob/rewrite_youtube_embed-3/shared/js/video/bundle.js#L25-L44)